recipes/fenics-ffcx/build-ufcx.sh

```diff
@@ -0,0 +1,6 @@
+cmake \
+  -DCMAKE_INSTALL_PREFIX="${PREFIX}" \
+  -B build-dir \
+  -S cmake/
+cmake --build build-dir
+cmake --install build-dir
```

Comment:
Is there a `make make` or `ctest` we can run here?

Reply:
`cmake --build` does the make. The test is run in `test-ufcx.sh`, which builds a cmake project against the ufcx.h